process/block/preprocess/transactions.go

```diff
@@ -23,8 +23,6 @@ import (
 
 var log = logger.GetOrCreate("process/block/preprocess")
 
-// TODO: increase code coverage with unit tests
```

Comment:
this file still has only around 50% coverage

Reply:
is 86.5% 
![image](https://user-images.githubusercontent.com/50499646/74641581-38430e00-517a-11ea-97a1-3153803a6870.png)

Reply:
Yeah, I was running only `transactions_test.go`. Are the unit tests from the other files relevant for this one?

Reply:
i  think you right maybe on another PR will be added more unit tests for transactions.go i will put // TODO: increase code coverage with unit tests where it was because there are not enough unit tests